### PR TITLE
#171380736 Integrating HoundCI and a Badge

### PR DESCRIPTION
**Pull request by ricahcyuzuzo**
#### What does this PR do?
Integrating HoundCI and add a badge in the readme.md file
#### Description of Task to be completed?
Have Hound CI Integrated on this repo and a badge in a readme file
#### How should this be manually tested?
After Cloning this repo, cd into it and raise a PR.
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#171380736](https://www.pivotaltracker.com/n/projects/2435482/stories/171380736)

### CODE REVIEW

**Comment by Muhire-Josue** [comment]
Looks good to me

**Comment by karamuka** [comment]
@ricahcyuzuzo, since you've changed this PR's label from **WIP** to **ready for review**, you can omit the **. but it's still in progress!** part of **How should this be manually tested?** section

**Comment by since9629** [comment]
@ricahcyuzuzo, in the `How should this be manually tested?` section, what does this line mean

> After Cloning this repo, cd into it and raise a PR.

**Comment by ricahcyuzuzo** [comment]
> @ricahcyuzuzo, in the `How should this be manually tested?` section, what does this line mean
> 
> > After Cloning this repo, cd into it and raise a PR.

@karamuka, Means to test is to clone the PR and go into the cloned repo, and raise a PR to test if it working!

File: .hound.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+javascript:
+  enabled: false
```
Comment (since9629) [inline .hound.yml:2]: @ricahcyuzuzo why did you disable JavaScript here?

File: .hound.yml [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+eslint:
+  enabled: true
+  config_file: .eslintrc.json
```
Comment (since9629) [inline .hound.yml:5]: you are referencing a non-existing file `.eslintrc.json`

Comment (ricahcyuzuzo) [reply .hound.yml:5]: I am waiting for your work to be merged and it will work!

File: .hound.yml (hunk 1 above)
Comment (ricahcyuzuzo) [reply .hound.yml:2]: @karamuka For enablling ESLint styling guides only not javascript styling guides

**Review by Muhire-Josue — APPROVED** [review]
LGTM

**Comment by since9629** [comment]
Good job @ricahcyuzuzo, but since we've configured ESLint, could you format your code with it?

**Review by since9629 — APPROVED** [review]
Good job

**Review by descholar-ceo — APPROVED** [review]
LGTM

File: README.md [hunk 3]
```diff
@@ -1,10 +1,11 @@
-Barefoot Nomad - Making company travel and accomodation easy and convinient.
-=======
+# Barefoot Nomad - Making company travel and accomodation easy and convinient.
 
-[![Build Status](https://travis-ci.org/Stackup-Rwanda/stackup2-barefoot-backend.svg?branch=develop)](https://travis-ci.org/Stackup-Rwanda/stackup2-barefoot-backend)
 [![Coverage Status](https://coveralls.io/repos/github/Stackup-Rwanda/stackup2-barefoot-backend/badge.svg?branch=develop)](https://coveralls.io/github/Stackup-Rwanda/stackup2-barefoot-backend?branch=develop)
+[![Reviewed by Hound](https://img.shields.io/badge/Reviewed_by-Hound-8E64B0.svg)](https://houndci.com)
```
Comment (mifeille) [inline README.md:5]: Please put your badge in the last position. (travis, coverage, hound)

**Pull request merged**